internal/service/alertconfiguration/data_source_alert_configuration.go

```diff
@@ -20,22 +20,22 @@ import (
 var _ datasource.DataSource = &alertConfigurationDS{}
 var _ datasource.DataSourceWithConfigure = &alertConfigurationDS{}
 
-type tfAlertConfigurationDSModel struct {
+type TfAlertConfigurationDSModel struct {
```

Comment:
nit: just for consistency with how it is defined in search deployment, open to discussing if we want to change this
```suggestion
type TFAlertConfigurationDSModel struct {
```